modules/about-etcd-encryption.adoc

```diff
@@ -21,5 +21,7 @@ When you enable etcd encryption, encryption keys are created. These keys are rot
 
 [NOTE]
 ====
-Keep in mind that etcd encryption only encrypts values, not keys. This means that resource types, namespaces, and object names are unencrypted.
+Etcd encryption only encrypts values, not keys. Resource types, namespaces, and object names are unencrypted.
+
+If etcd encryption is enabled during a backup, the `__static_kuberesources_<datetimestamp>.tar.gz: file__` contains the encryption keys for the etcd snapshot. For security reasons, store this file separately from the etcd snapshot. However, this file is required to restore a previous state of etcd from the respective etcd snapshot.
```

Comment:
@asme30 

I hate to dig this up again, but this still looks wrong. I've never seen a filename of the form: static_kuberesources_<datetimestamp>.tar.gz: file, so I'm not sure this is correct? Or is it: static_kuberesources_<datetimestamp>.tar.gz?